qemu/tests/tcpreplay.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import re
+import os
+
+from virttest import error_context
+from virttest import data_dir
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Use tcpreplay to replay a pcap file, and check the guest is alive
+    1) copy tcpreplay from deps directory
+    2) compile the tcpreplay
+    3) copy target pcap file from deps directory
+    4) use the rcpreply to replay the pcap file to guest
+    5) check the guest is still alive, no bsod occrued
+
+    param test: test object
+    param params: test params
+    param env: test environment
+    """
+
+    def execute_host_cmd(host_cmd, timeout=60):
+        """
+        Execute the host_cmd on host, limited in timeout period
+
+        param host_cmd: the host_cmd to run on host
+        param timeout: the timeout for running this command
+        return: the cmd output. Raise test.error is cmd failed
+        """
+        logging.info("Executing host command: %s" % host_cmd)
+        status, output = process.getstatusoutput(host_cmd, timeout=timeout)
+        if status:
```

Comment:
I believe `process.run` could be used here to signal that process non-zero exit status.